packages/ui/src/components/editor/input/Group/index.tsx

```diff
@@ -122,23 +122,33 @@ export interface InputGroupProps {
   className?: string
   labelClassName?: string
   infoClassName?: string
+  childrenClassName?: string
 }
 
 /**
  * InputGroup used to render the view of component.
  */
-export function InputGroup({ children, info, label, className, labelClassName, infoClassName }: InputGroupProps) {
+export function InputGroup({
+  children,
+  info,
+  label,
+  className,
+  labelClassName,
+  infoClassName,
+  childrenClassName
+}: InputGroupProps) {
```

Comment:
let's rename `className` to `containerClassName` and `childrenClassName` to `className`
it will be easy to refactor this using find and replace